e2e/tests/room/03-roomactions.spec.js

```diff
@@ -442,13 +446,27 @@ describe('Room actions screen', () => {
 				};
 
 				it('should show all users', async () => {
-					await element(by.id('room-members-view-toggle-status')).tap();
+					await waitFor(element(by.id('room-members-view-filter')))
+						.toExist()
+						.withTimeout(10000);
+					await element(by.id('room-members-view-filter')).tap();
+					await waitFor(element(by.id('room-members-view-toggle-status-all')))
+						.toExist()
+						.withTimeout(2000);
+					await element(by.id('room-members-view-toggle-status-all')).tap();
 					await waitFor(element(by.id(`room-members-view-item-${user.username}`)))
 						.toExist()
 						.withTimeout(60000);
+					await tapBack();
 				});
 
 				it('should filter user', async () => {
+					await element(by.id('room-actions-members')).tap();
```

Comment:
Always `waitFor(...).toExist().withTimeout(2000)` before tapping stuff.
It makes tests more reliable.
The way it's right now, the test is a bit flaky, because it relies only on the `tapBack` from the previous test.
The `tapBack` triggers the pop animation from react-navigation and only then we can actually `tap` the item.